### PR TITLE
Slow type and run very long command in scripted way

### PR DESCRIPTION
**Pull request by mitiao**
It aims to avoid 'key event queue full' issue:
https://openqa.suse.de/tests/1020607

For the detail, see:
https://progress.opensuse.org/issues/19540

Verified run:
http://147.2.207.208/tests/603

### CODE REVIEW

**Comment by coolo** [comment]
looks good, but I wouldn't make it arch specific. The type_string is already slower on power, so switching to current_script for all archs would be more consistent in reviewing/error handling

**Comment by mitiao** [comment]
Ok, hope that it would not spent long time on checking dozen repo urls with multiple addons registered :)

File: lib/utils.pm [hunk 1]
```diff
@@ -66,6 +66,7 @@ our @EXPORT = qw(
   service_action
   assert_gui_app
   install_all_from_repo
+  slow_run_scripted_commnad
```
Comment (okurz) [inline lib/utils.pm]: s/comnad/command/, or maybe even better name: `run_scripted_command_slow` to sound similar to `type_string_slow`

Comment (mitiao) [inline lib/utils.pm]: that sounds good. naming is hard to me sometimes :)

**Comment by okurz** [comment]
LGTM

**Comment by okurz** [comment]
I assume https://openqa.suse.de/tests/1037812#step/zypper_lr/11 is fixed by https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3204 ?

**Comment by mitiao** [comment]
Yes, see: https://openqa.suse.de/tests/1038285#step/repos_check/10
Please re-trigger that

**Pull request merged**